Core/MIPS/MIPSTables.cpp

```diff
@@ -1057,6 +1057,10 @@ int MIPSInterpret_RunUntil(u64 globalTicks) {
 			// DEBUG_LOG(CPU, "Hit the max ticks, bailing 1 : %llu, %llu", globalTicks, CoreTiming::GetTicks());
 			return 1;
 		}
+#ifdef __EMSCRIPTEN__
+        //Never block the main loop... But this seems to cause the emulator to crash after a bit..?
```

Comment:
PPSSPP should normally do as much as it can until it gets a frame, which will bail out of this function.  Probably if you're seeing it never exit this function, it's some failure in processing graphics commands.  This could be related to the uncached mirror problem, since uncached memory accesses are used heavily by games to synchronize with the GPU (which doesn't see things in the CPU caches.)

-[Unknown]

Reply:
That makes sense. Thanks for your input. That should lead me somewhere